tests/lib/rules/no-typos.js

```diff
@@ -795,5 +806,29 @@ ruleTester.run('no-typos', rule, {
     }, {
       message: 'Typo in declared prop type: objectof'
     }]
+  }, {
+    code: `class Component extends React.Component {};
+     Component.propTypes = {
+       b: string.isrequired
+     }
+   `,
+    parser: 'babel-eslint',
```

Comment:
same with these 2 tests; please duplicate them so there's 1 for babel-eslint and 1 for the default parser.